### PR TITLE
New DTI Website: Use Details/Summary For FAQ Accordion

### PR DESCRIPTION
**Pull request by andrew032011**
### Summary 
For the FAQ accordion [here](https://github.com/cornell-dti/idol/blob/e6b082a57fab1e1e4b414934c169fb5ce5d2caf6/new-dti-website/components/course/DDProjects.tsx#L33), we should use the `<details>` and `<summary>` tags. See this [MDN article for details](https://developer.mozilla.org/en-US/blog/html-details-exclusive-accordions/).

### Notion/Figma Link 
Part of https://github.com/cornell-dti/idol/issues/734

### Test Plan 
Verify FAQ Accordion looks and behaves the same as before.

### CODE REVIEW

File: new-dti-website/src/app/globals.css
Comment (andrew032011) [inline new-dti-website/src/app/globals.css:220]: Clicking on the accordian a few times in a row leads to some of the summary text getting highlighted, which is a regression from previous behavior. So I made the details not selectable.

**Review by cchrischen — APPROVED** [review]
Looks good! Since we are standardizing the accordions between the apply page and the course page, we should check to see which design to settle on before merging this one, in case we settle on the courses one.

**Comment by andrew032011** [comment]
> > Looks good! Since we are standardizing the accordions between the apply page and the course page, we should check to see which design to settle on before merging this one, in case we settle on the courses one.
> 
> I'll surface this in #idol-design.

Vannessa brought up a good point that the Application and Courses Project Showcase Accordians are for different purposes, so it makes sense to make them different. So we decided not to standardize them.

File: new-dti-website/src/app/globals.css
Comment (Bookie0) [reply new-dti-website/src/app/globals.css:220]: I do get ur point tho andrew and it is a shame that using `<details>` leads to this regression, so if you feel like that adding `user-select: none;` is worth it in this case, I'll defer to you here.

but let's also make sure that this is an exception bc i dont think we want to make this behavior standard

**Review by Bookie0 — APPROVED** [review]
THanksss!

File: new-dti-website/src/app/globals.css
Comment (andrew032011) [reply new-dti-website/src/app/globals.css]: Yeah, I had a feeling it was bad practice. I reverted that part! I agree with you that rapidly clicking might be something we do when testing it, but in practice, we wouldn't do that so I wouldn't consider it to be a "primary task purpose of a control".

Comment (Bookie0) [reply new-dti-website/src/app/globals.css]: ok, sounds good then!

**Pull request merged**
